projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -5607,6 +5607,11 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
      * @internal
      */
     public copyHandler(event) {
+        if (event.path?.map(el => el.tagName).includes('IGX-GRID-FILTERING-ROW') ||
```

Comment:
`event.path` is non-standardized and is undefined in Firefox. Use `event.composedPath()` instead.

Reply:
Also you are mapping the path twice when you can only do it once. In addition, tagName being all upper case is not always true. If you google search it you'll find cases where it can be  lowercase. You should normalize the search by applying toLower/UpperCase in the mapping.